### PR TITLE
Add traffic manager and /get_client_status client API for traffic stats.

### PR DESCRIPTION
**Pull request by platfowner**
Change summary:
- Add TrafficDb, TrafficManager modules
- Add /get_client_status client API including traffic stats
- Rename: numAliveNodes -> numNodesAlive

Related issues:

### CODE REVIEW

**Comment by platfowner** [comment]
Just renamed modules:
- TrafficDB -> EventCounter
- TrafficManager -> TrafficStatsManager

**Review by liayoo — COMMENTED** [review]
Nice traffic stats manager! Do you think we could turn this feature on & off with a FeatureFlag? (Or just use LIGHTWEIGHT=true?)

**Comment by platfowner** [comment]
> Nice traffic stats manager! Do you think we could turn this feature on & off with a FeatureFlag? (Or just use LIGHTWEIGHT=true?)

Just added a flag: enableTrafficMonitoring.

File: p2p/index.js [hunk 1]
```diff
@@ -120,6 +130,21 @@ class P2pClient {
...
 
+  getClientStatus() {
+    return {
+      trafficStats: this.getTrafficStats(),
+    };
+  }
+
+  getTrafficStats() {
+    const stats = {};
+    for (const periodName of Object.keys(TRAFFIC_STATS_PERIOD_SECS_LIST)) {
```
[the remark this answers is not in the document]
Comment (platfowner) [reply p2p/index.js]: Done. Thanks!

File: traffic/event-counter.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
+const _ = require('lodash');
+
+class EventCounter {
+  constructor(
+      intervalMs = 60000,  // 1 min
+      maxIntervals = 180,
+      currentTimeMs = null) {  // 3 hours
```
[the remark this answers is not in the document]
Comment (platfowner) [reply traffic/event-counter.js]: Just fixed. Thanks.

**Comment by platfowner** [comment]
Comments addressed. PTAL~ @liayoo

**Review by liayoo — APPROVED** [review]
LGTM, thanks!

**Pull request merged**